lib/board.rb

```diff
@@ -72,16 +74,9 @@ def render_helper(starting_character, num_column,show_ship)
       end
       coord = coord.next 
     end
-    next_line.join(" ") + "\n" 
+    rendered_line = next_line.join(" ") + " \n" 
+    puts rendered_line
+    rendered_line
```

Comment:
Fixed the return value for proper render and passing test for  the render method.